src/renderer/interfaces/LightMetadata.ts

```diff
@@ -0,0 +1,13 @@
+import REGL = require('regl');
+
+export interface LightMetadata {
+    lightPositions: REGL.Vec3[];
+    lightColors: REGL.Vec3[];
+    lightIntensities: number[];
+}
+
+export const blankLightMetadata: LightMetadata = {
+    lightPositions: [0, 0, 0],
+    lightIntensities: [0, 0, 0],
```

Comment:
Intensity should just be a scalar instead of an array. Unfortunately it looks like typescript is bad at actually typechecking `REGL.anything` types

Reply:
yeah, my linter was complaining. Should it be `lightIntensities` or just `lightIntensity` then?